sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -1287,6 +1289,444 @@ public enum TextSentiment
         Mixed = 3,
     }
 }
+namespace Azure.AI.TextAnalytics.Models
+{
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct AgeUnit : System.IEquatable<Azure.AI.TextAnalytics.Models.AgeUnit>
```

Comment:
It seems structs are not being generated with the `internal` modifier by default. You may need to add those to `InternalTypes.cs` if you want to keep them internal for now.